### DISCUSSION TITLE
Use cases @ Workshop on December 10

### DISCUSSION THREAD

**Discussion by smrbu** [ProxeusApp/community#9] [Use case]
In our workshop on 10.12.2021, together with participants among our existing and interested users, we collected and prioritized 14 use cases for Proxeus using the How-Now-Wow matrix, to understand where how to position the most relevant and actionable ideas for the association. These are listed as individual comments below.

Please use the comment function to reply with any additional suggestions, thoughts, links, and support possibilities. With this feedback and encouragement, we will aim to drive the project and software to the next milestone :rocket: 

![Screenshot from 2021-12-10 09-52-08](https://user-images.githubusercontent.com/31819/212694851-12aaf165-7ced-4822-9d79-215ff4bc3a8a.jpg)

**Comment by smrbu** [comment]
Product certificates to **_verify ownership_** (Wow)

**Comment by smrbu** [comment]
**_No-code engine_** to design your NFTs (Wow)

**Comment by smrbu** [comment]
Company registry: **_on chain, in business_** (Wow / Now)

**Comment by loleg** [comment]
**_Prototype fund_** workflows for democracy (Wow)

**Comment by loleg** [comment]
Open platform for _**educational purposes**_ (Now / Wow)

**Comment by loleg** [comment]
_**Employee Stock Option Plan**_ that can be easily created without code and 'traded' (Now / Wow)

**Comment by loleg** [comment]
_**Certification**_ for data hackdays (Now)

**Comment by loleg** [comment]
Separation of _**build-time and run-time**_ (Now)

**Comment by loleg** [comment]
Connect built-environment with _**algorithmic architecture**_ (How)

**Reply by loleg** [reply]
[AlgoTecture](https://algotecture.net/)

**Comment by loleg** [comment]
_**One-click**_ open source automation (How)

**Reply by loleg** [reply]
Leverage the experience of making Proxeus [easier to deploy across clouds](https://github.com/ProxeusApp/community/discussions/3), see if we can inspire other projects or even help to design, deploy and verify open source deployments with Proxeus directly.

**Comment by loleg** [comment]
Fractional ownership, _**tokenization of energy**_ (How)

**Reply by loleg** [reply]
See the project [07 - DLT 4 financing the energy transition](https://hack.opendata.ch/project/685) ([Energy & Climate Hack](https://hack.opendata.ch/event/36) 2021) which goes into some detail about how this might work. We have looked at the Proxeus platform with several teams doing energy-related tech prototyping.

**Comment by loleg** [comment]
**_Data brokerage_** across stakeholders along the public/private sector (How)

**Comment by loleg** [comment]
_**Neighbourhood community**_ development & maintenance of new projects (How)

**Comment by loleg** [comment]
_**Measuring & certifying**_ contributions to projects, beyond software (How)
